model/sharing/files.go

```diff
@@ -322,7 +328,18 @@ func (s *Sharing) RemoveSharingDir(inst *instance.Instance) error {
 		Type: consts.Sharings,
 	})
 	dir.CozyMetadata.UpdatedAt = time.Now()
-	return inst.VFS().UpdateDirDoc(olddoc, dir)
+	suffix := inst.Translate("Tree Revoked sharing suffix")
+	parentPath := filepath.Dir(dir.Fullpath)
+	basename := fmt.Sprintf("%s (%s)", dir.DocName, suffix)
+	dir.DocName = basename
+	for i := 2; i < 100; i++ {
+		dir.Fullpath = path.Join(parentPath, dir.DocName)
+		if err = inst.VFS().UpdateDirDoc(olddoc, dir); err == nil {
+			return nil
+		}
+		dir.DocName = fmt.Sprintf("%s (%d)", basename, i)
```

Comment:
You set a limit to `(99)` for the docName's conflict? 

If I already have a folder called `Cozy-sharing (cancelled sharing)`, you will generate `Cozy-sharing (cancelled sharing) (2)` until 99 (if conflict) right? 

We have this kind of name generation on conflict on the front side, I'll use the same limit.

Reply:
Yes, you are right